src/views/preview/preview.jsx

```diff
@@ -13,7 +15,10 @@ const UnsupportedBrowser = require('./unsupported-browser.jsx');
 if (isSupportedBrowser()) {
     const ProjectView = require('./project-view.jsx');
     render(
-        <ProjectView.View />,
+        <div>
+            <TermsOfServiceModal />
+            <ProjectView.View />
```

Comment:
Project page returns either `PreviewPresentation` wrapped in `Page` or `IntlGUIWithProjectHandler`. Probably we can add `TermsOfServiceModal` only to `IntlGUIWithProjectHandler`